### DISCUSSION TITLE
Feature request: position of the word "Thorium" in the window title

### DISCUSSION THREAD

**Discussion by llemeurfr** [edrlab/thorium-reader#1748] [Ideas]
A user noticed: 

It would be great if the Thorium window title would put ‘Thorium’ after the book title, rather than before it. At the moment it makes it look like every book title is ‘Thorium – [Title]’ which actually does sound like a book!

This is open to discussion ...

**Comment by danielweck** [comment]
I totally understand this ... on the other hand, as a user myself of other applications that put their name at the end of the title, I find it really annoying not to be able to identify the app "at a glance" when switching windows. I anticipate that changing the current title structure would confuse existing users. We could add an option in the settings tab, but too much configuration also contributes to confusion!

**Comment by llemeurfr** [comment]
On MacOS apps act differently on this regard: 
- some  display nothing: Calendar, Safari, Mail, Chrome, Spotify ... 
- some only display the name of the current document: Pages, MS Word, MS Excel, **iBooks**
- some display contextual information: Terminal
- I found none having a display equivalent to Thorium

Waiting for testimonials on Windows.

**Comment by nandakishore2009** [comment]
> I totally understand this ... on the other hand, as a user myself of other applications that put their name at the end of the title, I find it really annoying not to be able to identify the app "at a glance" when switching windows. I anticipate that changing the current title structure would confuse existing users. We could add an option in the settings tab, but too much configuration also contributes to confusion!

You could add an option for enabling advance configuration, by that we could get choose to hide the advance customization menu items from novice users.

**Reply by llemeurfr** [reply]
I agree with Daniel, too many configuration param, even if filtered for novice users, makes the UX difficult (just compare Android vs iOS settings).

**Comment by llemeurfr** [comment]
Still waiting for testimonials on Windows and Linux.
